Testsuite: net.sourceforge.czt.animation.eval.AnimateScopeTest
Tests run: 18, Failures: 18, Errors: 0, Time elapsed: 0.033 sec

------------- Standard Output ---------------
visitRefExpr:S1 gives : [] net.sourceforge.czt.z.impl.ExistsExprImpl@555a1683
visitRefExpr:S1 --> net.sourceforge.czt.z.impl.ExistsExprImpl@555a1683
------------- ---------------- ---------------
Testcase: animate_scope.tex:22(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:23(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:26(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:27(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex::5(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.RefExprImpl@dfc732bd
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:34(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@be0ce40d
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@be0ce40d
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:35(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@86a6e506
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@86a6e506
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:41(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.ExistsPredImpl@ff9eda4c
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.ExistsPredImpl@ff9eda4c
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:44(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.ExistsPredImpl@4db3d8df
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.ExistsPredImpl@4db3d8df
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:49(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@be0ce40d
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@be0ce40d
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:50(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@593552e4
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@593552e4
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:52(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@4db32cd9
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@4db32cd9
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:53(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@fd1579a9
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.LetExprImpl@fd1579a9
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:57(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@a549951c
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@a549951c
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:58(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@e54dc683
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@e54dc683
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:59(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@e07866c8
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@e07866c8
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:60(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@11fb973e
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.Exists1PredImpl@11fb973e
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


Testcase: animate_scope.tex:70(net.sourceforge.czt.animation.eval.EvalTest$PredTest):	FAILED
Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.ExistsExprImpl@555a1683
junit.framework.AssertionFailedError: Should not throw exception java.lang.RuntimeException: Flatten does not yet handle: net.sourceforge.czt.z.impl.ExistsExprImpl@555a1683
	at net.sourceforge.czt.animation.eval.EvalTest$PredTest.runTest(EvalTest.java:107)


